bingo.js

```diff
@@ -435,6 +451,21 @@ function changeColourCount(value)
 	pushNewLocalSetting(COLOUR_COUNT_SETTING_NAME, COLOURCOUNT);
 }
 
+function updateColourSymbols()
+{
+	$(".symbol").css("display", COLOURSYMBOLS == 0 ? "none" : "inline");
```

Comment:
`COLOURSYMBOLS` is always either `true` or `false`. Comparison to zero is not needed.

Reply:
I had it as a number before, I guess I missed that one.